### DISCUSSION TITLE
How to change git commit username and email change for origin project

### DISCUSSION THREAD

**Discussion by rove-swl** [community/community#112431] [Programming Help]
### Select Topic Area

Question

### Body

How can I change username and email of old commit when I made wrong username and email for commit?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @aldemir9210 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by anvar-saidov-sh** [comment]
Open your terminal and navigate to your git repository. Change Git user name by running: git config --global user.name “your username”. Change Git user email by running: git config --global user. email “name@email.com”
